.buildkite/release-pipeline.yml

```diff
@@ -0,0 +1,114 @@
+## 🏠/.buildkite/pipeline-release.yml
```

Comment:
this may be out of scope, but to me it looks like the images we are going to publish here will contain hundreds of CVEs

it could be nice to see if we can invest the effort up-front to start by publishing [Wolfi-based images](https://docs.elastic.dev/wolfi), instead

Reply:
I think this sounds like a good idea. I'll try and switch this to Wolfi today.

Reply:
I had an attempt at this today but I'm running into various issues. I made a draft PR: https://github.com/elastic/crawler/pull/106
It doesn't work yet, but we can iterate on it.

Reply:
nice! I added a thought over there https://github.com/elastic/crawler/pull/106#pullrequestreview-2257621613